packages/dnb-eufemia/src/extensions/payment-card/types.ts

```diff
@@ -0,0 +1,108 @@
+import { SpacingProps } from '../../shared/types'
+
+export type PaymentCardProps = {
+  /**
+   * <em>(required)</em> if product code matches one of the codes in the list the card will get that design, if no match is found Default design will be used.
+   */
+  productCode?: string
+  /**
+   * <em>(required)</em> masked card number.
+   */
+  cardNumber?: string
+  /**
+   * Use one of these: `active`, `not_active`, `blocked`, `expired`, `renewed`, `replaced`, `order_in_process`, `unknown`. Defaults to `active`.
+   */
+  cardStatus?: PaymentCardCardStatus
+  /**
+   * Defines the appearance. Use one of these: `normal` or `compact`. Defaults to `normal`.
+   */
+  variant?: PaymentCardVariant
+  /**
+   * Will use 8 digits if none are specified.
+   */
+  digits?: number
+  /**
+   * Useful if you want to create custom cards. See Card data properties.
+   */
+  customCard?: CustomCard
```

Comment:
I think the prop name `customCard` is fitting and correct.
However I think the type could perhaps just be called `PaymentCard` og perhaps `PaymentCardType`, as the type is used to define `PaymentCard`'s, like in [/payment-card/utils/CardDesigns.ts](https://github.com/dnbexperience/eufemia/pull/3625/files#diff-490a4bfae3a612fb3ab5a74e1ab228a06411fda79128b147c2ba6e47a7984200) where we define all cards.

```suggestion
  customCard?: PaymentCard
```

```suggestion
  customCard?: PaymentCardType
```